scss/_patterns_side-navigation.scss

```diff
@@ -19,12 +19,12 @@
       @extend %vf-pseudo-border--top;
 
       &::after {
-        left: $sph-inner;
+        @include vf-side-navigation-spacing-left($prop: left);
```

Comment:
@bartaz so the mixin default is padding-left but this sets left instead?

Reply:
Exactly, because in most cases it needs to adjust `padding-left` (for the links), but in some cases (icon, border) it adjust `left` absolute position.